frameworks/Haskell/yesod-postgres/bench/.gitignore

```diff
@@ -0,0 +1,8 @@
+dist*
+static/tmp/
+config/client_session_key.aes
+*.hi
+*.o
+*.sqlite3
+.hsenv*
+yesod-devel/
```

Comment:
This file should be merged with `frameworks/Haskell/yesod-postgres/.gitignore`.

Additionally, is there a reason to not simply add `bench` to `.gitignore`?